CodeSnippetsReflection.OpenAPI/LanguageGenerators/GoGenerator.cs

```diff
@@ -0,0 +1,290 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+using System.Text.Json;
+using System.Text.RegularExpressions;
+using CodeSnippetsReflection.StringExtensions;
+using Microsoft.OpenApi.Models;
+using Microsoft.OpenApi.Services;
+
+namespace CodeSnippetsReflection.OpenAPI.LanguageGenerators {
+    public class GoGenerator : ILanguageGenerator<SnippetModel, OpenApiUrlTreeNode>
+    {
+        private const string clientVarName = "graphClient";
+		private const string clientVarType = "GraphServiceClient";
+		private const string httpCoreVarName = "requestAdapter";
+        public string GenerateCodeSnippet(SnippetModel snippetModel)
+        {
+            var indentManager = new IndentManager();
+			var snippetBuilder = new StringBuilder(
+									"//THE GO SDK IS IN PREVIEW. NON-PRODUCTION USE ONLY" + Environment.NewLine +
+									$"{clientVarName} := msgraphsdk.New{clientVarType}({httpCoreVarName});{Environment.NewLine}{Environment.NewLine}");
```

Comment:
```suggestion
									$"{clientVarName} := msgraphsdk.New{clientVarType}({httpCoreVarName}){Environment.NewLine}{Environment.NewLine}");
```
Do we really need the semicolon before the newlines? 
If not, I think we might be better off removing to be consistent with the rest of the generated code?